templates/_auth_ldap.erb

```diff
@@ -14,13 +14,13 @@ provider_url =
   providerUrl="<%= provider_url %>"
   authenticationMethod="simple"
   forceBindingLogin="<%= @auth_config['ldap']['force_binding'] %>"
-<%- if @auth_config['ldap']['force_binding_use_root'] != :undef -%>
+<%- if !@auth_config['ldap']['force_binding_use_root'].nil? && @auth_config['ldap']['force_binding_use_root'] != :undef -%>
```

Comment:
Considering `force_binding_use_root` [defaults to `false`](https://github.com/voxpupuli/puppet-rundeck/blob/2fdbe3e6999809249659620258897497f4276c4d/manifests/params.pp#L185) and not `undef`, I'm not convinced this conditional is needed at all.

Reply:
@fe80 I've taken the liberty of pushing a commit onto your branch (something you _can_ turn off when you open a PR) that removes this conditional and adds a test for this ldap property.  Hoping you could follow the pattern and add tests for the other properties you've fixed?  Feel free to discard/squash it if you want to add more tests or write them in a different way. :)